cypress/e2e/banUser.cy.js

```diff
@@ -1,7 +1,6 @@
 describe('Ban User Form', () => {
     beforeEach(() => {
         cy.cleanDB()
-        cy.populateWithDrivers()
```

Comment:
If you don't populate DB with Users how can you ban anyone? Does this test pass?